src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -303,6 +306,46 @@ public static JsonArrayBuilder json(List<DataverseContact> dataverseContacts) {
         }
         return jsonArrayOfContacts;
     }
+    
+    public static JsonArrayBuilder getOwnersFromDvObject(DvObject dvObject) {
+        
+        List <DvObject> ownerList = new ArrayList();
```

Comment:
I've noticed a potential improvement in the data structure format we're using. The current implementation may not intuitively indicate the hierarchical relationship between "root" and "First Dataset" unless one already understands the intended structure. This could lead to misunderstandings, especially for those unfamiliar with the context.

Here's the current payload structure:
```javascript
ownerArray = [
  {
    type: 'DATASET',
    identifier: 'doi:10.5072/FK2/WTBMGC',
    displayName: 'First Dataset'
  },
  { type: 'DATAVERSE', identifier: 'root', displayName: 'Root' }
];
```
To enhance clarity, I suggest adopting a nested object approach, where each entity has an 'owner' property.

Proposed payload structure:
```javascript
ownerArray = {
  type: 'DATASET',
  identifier: 'doi:10.5072/FK2/WTBMGC',
  displayName: 'First Dataset',
  owner: {
    type: 'DATAVERSE',
    identifier: 'root',
    displayName: 'Root'
  }
};
```
I'd appreciate your thoughts on this proposed adjustment.

Reply:
I also think the structure @MellyGray mentions would improve readability and make the response more intuitive and self-explanatory.

Reply:
also - using "isPartOf" to mirror https://schema.org/isPartOf might be better than "owner" - that's what is used in the OREMap file.